hack/kwok-perftests.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+#
+# Copyright 2024 The KubeVirt Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This script should be executed through the makefile via `make kwok-perftest`.
+
+set -e
+
+kubectl() { KUBEVIRTCI_VERBOSE=false cluster-up/kubectl.sh "$@"; }
+
+DOCKER_TAG=${DOCKER_TAG:-devel}
+DOCKER_TAG_ALT=${DOCKER_TAG_ALT:-devel_alt}
+PROMETHEUS_PORT=${PROMETHEUS_PORT:-30007}
+
+source hack/common.sh
+source hack/config.sh
+
+_default_previous_release_registry="quay.io/kubevirt"
+
+previous_release_registry=${PREVIOUS_RELEASE_REGISTRY:-$_default_previous_release_registry}
+
+perftest_docker_prefix=${manifest_docker_prefix-${docker_prefix}}
+
+if [[ ${KUBEVIRT_PROVIDER} == os-* ]] || [[ ${KUBEVIRT_PROVIDER} =~ (okd|ocp)-* ]]; then
+    oc=${kubectl}
+fi
+
+if [ -z "$kubeconfig" ]; then
+    kubeconfig="$KUBECONFIG"
+fi
+
+echo 'KWOK Performance testing'
+export KUBEVIRT_E2E_KWOK_PERF_TEST=true
+
+echo 'Create kubevirt related kwok manifests'
+kubectl create -k tests/performance/manifests/kwok
```

Comment:
nit: change this to `kubectl apply -k`

In the case of local testing, create breaks the test if manifests are already present, apply will continue to work even if manifest are present